R/fair-demographic_parity.R

```diff
@@ -0,0 +1,47 @@
+#' Demographic parity
```

Comment:
In this and the other metrics, I like what you are doing with the templating. I still think it would be good to make sure we include a one-liner describing what the function does. e.i.

> demographic parity is calculated as the difference between the largest and smallest value of detection_prevalence that were calculated for each group

Reply:
Addressed in [a2dfe36](https://github.com/tidymodels/yardstick/pull/434/commits/a2dfe363a9fed26c5f204f9c87237d03f957df0b)!